authentication/src/main/java/com/ecommercepractice/authentication/util/Pair.java

```diff
@@ -0,0 +1,21 @@
+package com.ecommercepractice.authentication.util;
+
+import lombok.Getter;
+
+/**
+ * Tuple of key value [key,Value], it is used when sending
+ * the invalid fields onto the user.
+ * @param <T>
+ * @param <G>
+ */
+@Getter
+public class Pair<T,G>  {
+    private T key;
+    private G value;
+
```

Comment:
Use Lombok t avoid boileplate code

Reply:
I think you are talking about the constructor with all args, so I remove it and put @AllArgsConstructor